From 3010a30dcc0ac493fcfe485a8ad1ff483fa0a01d Mon Sep 17 00:00:00 2001
From: Daniel Mack <daniel@zonque.org>
Date: Mon, 1 Jun 2015 12:06:07 +0200
Subject: [PATCH] zsh-completion: fix typo in _bootctl

The command is 'install', not 'instal'. Fix that typo.

(cherry picked from commit c521a430fd6027d55f96516bc2f7570f5997e137)
---
 shell-completion/zsh/_bootctl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shell-completion/zsh/_bootctl b/shell-completion/zsh/_bootctl
index ce776c052f..0e1b0a5562 100644
--- a/shell-completion/zsh/_bootctl
+++ b/shell-completion/zsh/_bootctl
@@ -5,7 +5,7 @@
     local -a _bootctl_cmds
     _bootctl_cmds=(
         "status:Show status of installed systemd-boot and EFI variables"
-        "instal:Install systemd-boot to the ESP and EFI variables"
+        "install:Install systemd-boot to the ESP and EFI variables"
         "update:Update systemd-boot in the ESP and EFI variables"
         "remove:Remove systemd-boot from the ESP and EFI variables"
     )
